.travis.yml

```diff
@@ -0,0 +1,36 @@
+language: objective-c
+osx_image: xcode8.3
```

Comment:
Just FYI, http://lint.travis-ci.org/ says "*osx_image* is experimental and might be removed in the future".